src/app/docs/split-view/split-view-docs.component.ts

```diff
@@ -2,8 +2,162 @@ import {
   Component
 } from '@angular/core';
 
+import {
+  FormControl,
+  FormGroup
+} from '@angular/forms';
+
+import {
+  Subject
+} from 'rxjs';
```

Comment:
In the current code, we say `from 'rxjs/Subject';`. Is the change correct?

Reply:
Yup. That was a holdover from earlier Angular/RxJs versions that hadn't been updated in the demo.